### ISSUE TITLE
 Multi label problems

### ISSUE THREAD

**Issue by sonacharles** [Superzchen/iLearnPlus#10]
While performing multi label problems, all performance evaluation matrices are showning NA except Accuracy. ROC/ PRC is also not generated. Kindly help.

**Comment by Superzchen** [comment]
Yes, that 's right. When performint multi label problems, only the accuracy be calculated due to the other metrics is designed for binary classification task.
